### ISSUE TITLE
`!:` prevents tester from running tests in file

### ISSUE THREAD

**Issue by belisarius222** [urbit/arvo#675]
Adding a `!:` to the top of a test file causes the file's tests not to be run.

**Comment by ohAitch** [comment]
Like, above the `/+`? That sounds like a syntax error yeah

**Comment by ohAitch** [comment]
As per #655, this should definitely be "test reports as unbuilt" and not "test fails to run" however.

**Comment by belisarius222** [comment]
Oh, you can't put a `!:` above a `/+`?

**Comment by belisarius222** [comment]
Because Ford runes have to come first?

**Comment by cgyarvin** [comment]
Unfortunately !: is not a valid ford rune.  This is certainly a misfeature...

**Comment by ohAitch** [comment]
Yeah, it would make sense for it to look for the rune and toggle it for the whole file, but right now it's on by default instead.

**Comment by ohAitch** [comment]
Ford runes as a syntactically separate construct are a misfeature, they should just have `constexpr` type restrictions instead imo.
